pygsti/objects/povm.py

```diff
@@ -1075,6 +1080,63 @@ def __str__(self):
             % (self.nqubits, str(self.qubit_filter))
         return s
 
+class CHPPOVM(POVM):
+    """Temporary POVM for CHP.
+
+    """
+    def __init__(self, nqubits, evotype='chp', error_op=None):
+        """Create a new CHPPOVM object.
+
+        Parameters:
+        ----------
+        nqubits: int
+            Number of qubits to measure
+        
+        evotype: str
+            Must be "chp" for now
+
+        error_op: LinearOperator, optional
+            An operator to act on before the measurement.
+            Must either act on all qubits or one qubit
+        """
+        self.nqubits = nqubits
+        assert evotype == 'chp', "Must be 'chp' ev"
+    
+    def simplify_effects(self, prefix=""):
+        """
+        Creates a dictionary of simplified effect vectors.
+
+        Returns a dictionary of effect SPAMVecs that belong to the POVM's parent
+        `Model` - that is, whose `gpindices` are set to all or a subset of
+        this POVM's gpindices.  Such effect vectors are used internally within
+        computations involving the parent `Model`.
+
+        Parameters
+        ----------
+        prefix : str
+            A string, usually identitying this POVM, which may be used
+            to prefix the simplified gate keys.
+
+        Returns
+        -------
+        OrderedDict of SPAMVecs
+        """
+        # TODO: CHP short circuit
+        if self._evotype == 'chp':
```

Comment:
Shouldn't this always be True given __init__?

Reply:
Removed unused CHPPOVM